haystack/nodes/prompt/prompt_model.py

```diff
@@ -78,6 +80,7 @@ def create_invocation_layer(
     ) -> PromptModelInvocationLayer:
         kwargs = {
             "api_key": self.api_key,
+            "api_base": self.api_base,
```

Comment:
Just add this key iff `self.api_base` is not `None`.

Reply:
OK
`
        kwargs = {
            "api_key": self.api_key,
            "timeout": self.timeout,
            "use_auth_token": self.use_auth_token,
            "use_gpu": self.use_gpu,
            "devices": self.devices,
        }
        if self.api_base is not None:
            kwargs["api_base"] = self.api_base
`